### PR TITLE
Ipp usb feature

### PR DESCRIPTION
**Pull request by tjamhar**
Initial review commit

### CODE REVIEW

**Comment by tjamhar** [comment]
@alexke-dev is added to the review. #Closed

**Comment by tjamhar** [comment]
@mziller is added to the review. #Closed

**Comment by tjamhar** [comment]
@jlsancam is added to the review. #Closed

**Comment by tjamhar** [comment]
@mziller is added to the review. #Closed

**Comment by tjamhar** [comment]
@alexke-dev is added to the review. #Closed

File: epan/dissectors/packet-ippusb.c [hunk 1]
```diff
@@ -0,0 +1,100 @@
+/* packet-ippusb.c
+ * Routines for IPPUSB packet disassembly
+ *
+ * Jamie Hare <jamienh@umich.edu>
```
Comment (alexke-dev) [inline epan/dissectors/packet-ippusb.c:4]: ><jamienh@umich.edu> [](start = 14, length = 19)

I would probably switch this to MSFT email?

Comment (tjamhar) [reply epan/dissectors/packet-ippusb.c:4]: My microsoft email will be deleted at the end of my internship

---
In reply to: [442546781](https://github.com/tjamhar/wireshark/pull/1#discussion_r442546781) [](ancestors = 442546781)

**Review by alexke-dev — APPROVED** [review]
:shipit:

**Pull request merged**